### PR TITLE
add tpv34 benchmark

### PR DESCRIPTION
**Pull request by git-taufiq**

### CODE REVIEW

File: tpv34/ASAGI_file [hunk 1]
```diff
@@ -0,0 +1,6 @@
+tpv34_mux-fault.nc: 
+https://syncandshare.lrz.de/getlink/fiPzx6ebB4oor4BjtXvpmM92/tpv34_mux-fault.nc
```
Comment (Thomas-Ulrich) [inline tpv34/ASAGI_file:2]: This is not good as it is not a permanent link.
Would be better to point to the script you use to generate these files

File: tpv34/parameters.par [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+printtimeinterval_sec = 1.0                    ! Time interval at which output will be written
+OutputMask = 1 1 1 1 1 1 1 1 1 1 1             ! turn on and off fault outputs
+refinement_strategy = 2
+refinement = 1
+/
+
+! parameterize ascii fault file outputs
+&Pickpoint
+printtimeinterval = 1       ! Index of printed info at timesteps
+OutputMask = 1 1 1 0        ! turn on and off fault outputs
```
Comment (Thomas-Ulrich) [inline tpv34/parameters.par:49]: I think you need more 1 in the OutputMask.

File: tpv34/tpv34_fault.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+!Switch
+[mu_d, mu_s, d_c]: !ConstantMap
+  map:
+    mu_d: 0.45
+    mu_s: 0.58
+    d_c: 0.18
+[cohesion]: !FunctionMap
+  map:
+    cohesion: |
+      return 0.0-425.0*max(z+2400.0,0.0);
```
Comment (Thomas-Ulrich) [inline tpv34/tpv34_fault.yaml:10]: remove 0-

File: tpv34/ASAGI_file (hunk 1 above)
Comment (git-taufiq) [reply tpv34/ASAGI_file:2]: it is quite a big file and need some time to create it. any idea where to put it then?
I will provide a script on the tpv34 cookbook (ongoing).

File: tpv34/tpv34_material.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+!Switch
+[rho, mu, lambda]: !IdentityMap
+  components:
+    - !AxisAlignedCuboidalDomainFilter
```
Comment (Thomas-Ulrich) [inline tpv34/tpv34_material.yaml:4]: comment why you need 2 netcdf here.

Comment (git-taufiq) [reply tpv34/tpv34_material.yaml:4]: right, will add comments there

File: tpv34/tpv34_fault.yaml (hunk 3 above)
Comment (git-taufiq) [reply tpv34/tpv34_fault.yaml:10]: you mean remove 0.0 only? I think the negative is still needed

Comment (Thomas-Ulrich) [reply tpv34/tpv34_fault.yaml:10]: yes

File: tpv34/ASAGI_file (hunk 1 above)
Comment (Thomas-Ulrich) [reply tpv34/ASAGI_file]: How much time? Why does it take time to generate it?

Comment (git-taufiq) [reply tpv34/ASAGI_file]: It depends on the number of input points for CVM-H. They have a 1km resolution. 
Anyway, I removed the links already and just directing it to the tpv34 cookbook, see https://github.com/SeisSol/Examples/pull/5/commits/c4dd7d3bab35bf83a163817f2051bca6070958d6

**Comment by Thomas-Ulrich** [comment]
Hi, 
reviewer approval was not enabled for pull requests on this repository.
I've turned it on.
Next time, ask me when you have done the proposed change before merging!

**Comment by Thomas-Ulrich** [comment]
But it looks fine! Thx.

**Pull request merged**